src/app.py

```diff
@@ -0,0 +1,127 @@
+import json
+import logging
+import os
+import uuid
+from typing import Tuple
+
+from flask import Flask, request, jsonify, Response, send_file
+
+import csv
+from analyzer_engine.csv_analyzer_engine import CSVAnalyzerEngine
+from presidio_analyzer import DictAnalyzerResult, RecognizerResult
+from presidio_anonymizer import BatchAnonymizerEngine
+from config.nlp_engine_config import FlairNLPEngine
+
+DEFAULT_PORT = "3000"
+NLP_ENGINE = "flair/ner-english-large"
+UPLOAD_DIR = "file_uploads"
+
+class Server:
+    """HTTP Server for calling Presidio Analyzer."""
+
+    def __init__(self):
+        self.logger = logging.getLogger("pii-detection-anonymizer")
+        self.logger.setLevel(os.environ.get("LOG_LEVEL", self.logger.level))
+        self.app = Flask(__name__)
+        self.logger.info("Starting analyzer engine")
+        nlp_engine = FlairNLPEngine(NLP_ENGINE)
```

Comment:
We shld also think about how model loading can be done efficiently. For every request it shld not download it.